plugins/sosreport/skills/scripts/sosreport-multi-collector.py

```
@@ -0,0 +1,634 @@
+            self.safe_print(f"    /sosreport:analyze {self.download_dir}/sosreport-*.tar.xz")
+
+        if failures > 0:
+            self.safe_print(f"  • Check logs for failures:")
+            self.safe_print(f"    cat {self.logs_dir}/<node-name>-*.log")
+            self.safe_print(f"  • Retry failed nodes individually:")
+            for node in list(failed_nodes.keys())[:2]:  # Show first 2
+                short_node = node.split('.')[0]
+                self.safe_print(f"    /sosreport:collector all {node}")
+
+        if not self.auto_cleanup and pods_created > 0:
+            self.safe_print(f"  • Cleanup debug pods:")
```

Ruff (0.14.6)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)